examples/default/src/screens/CrashReportingScreen.tsx

```diff
@@ -46,6 +50,48 @@ export const CrashReportingScreen: React.FC = () => {
     }
   }
 
+  const styles = StyleSheet.create({
+    inputWrapper: {
+      padding: 4,
+      flex: 1,
+    },
+
+    formContainer: {
+      flexDirection: 'row',
+      alignItems: 'stretch',
+    },
+  });
```

Comment:
We should move these `styles` outside the component function to not get recreated with each render. We usually define it under the component function.